containers/fedora/user-data

```diff
@@ -20,11 +20,27 @@ write_files:
 
         [Install]
         WantedBy=cloud-init.service
+packages:
```

Comment:
Is this plugin taking care of cleaning up the cache?

> Architecture-specific packages can now be installed in a separate step, enhancing flexibility and maintainability.

I remember reviewing a change that had a condition on what is installed.
How will this be possible now? Can you give an example?

Reply:
> Is this plugin taking care of cleaning up the cache?
I think yes, but I'm not entirely certain.

> I remember reviewing a change that had a condition on what is installed.
> How will this be possible now? Can you give an example?
The common packages will be installed in this manner, while the architecture-specific packages will be installed using the dnf command in runcmd. Since we anticipate that the installation may fail on other architectures, we can use the following structure: `dnf install package -y || true`

Reply:
Hey @EdDev can we resolve this comment? Any opinion on this matter?

Reply:
> > Is this plugin taking care of cleaning up the cache?
> 
> I think yes, but I'm not entirely certain.

Can you just check if the image size grow or not? That will be a sign.

> we can use the following structure: `dnf install package -y || true`

That will hide a problem and we will find out about it too late.
A condition or a layered build would work for me.

If the size diff is good, you can resolve this thread.

Reply:
@EdDev, lines 26 and 27 still handle the autoremove and clean all packages. Do you think verification is still necessary?
As for the `dnf install -y || true`, we can also use an if statement if you prefer. The key is to ensure that the solution remains clean and understandable for everyone.